packages/lending/src/protocols/spark/tokens/data/mainnet.json

```diff
@@ -0,0 +1,121 @@
+{
+  "ETH": {
+    "chainId": 1,
+    "address": "0x0000000000000000000000000000000000000000",
+    "decimals": 18,
+    "symbol": "ETH",
+    "name": "Ethereum"
+  },
+  "DAI": {
+    "chainId": 1,
+    "address": "0x6B175474E89094C44Da98b954EedeAC495271d0F",
+    "decimals": 18,
+    "symbol": "DAI",
+    "name": "Dai Stablecoin"
+  },
+  "spDAI": {
+    "chainId": 1,
+    "address": "0x4DEDf26112B3Ec8eC46e7E31EA5e123490B05B8B",
+    "decimals": 18,
+    "symbol": "spDAI",
+    "name": "Spark DAI"
+  },
+  "sDAI": {
+    "chainId": 1,
+    "address": "0x83F20F44975D03b1b09e64809B757c47f942BEeA",
+    "decimals": 18,
+    "symbol": "sDAI",
+    "name": "Savings Dai"
+  },
+  "spsDAI": {
+    "chainId": 1,
+    "address": "0x78f897F0fE2d3B5690EbAe7f19862DEacedF10a7",
+    "decimals": 18,
+    "symbol": "spsDAI",
+    "name": "Spark sDAI"
+  },
+  "USDC": {
+    "chainId": 1,
+    "address": "0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48",
+    "decimals": 6,
+    "symbol": "USDC",
+    "name": "USD Coin"
+  },
+  "spUSDC": {
+    "chainId": 1,
+    "address": "0x377C3bd93f2a2984E1E7bE6A5C22c525eD4A4815",
+    "decimals": 6,
+    "symbol": "spUSDC",
+    "name": "Spark USDC"
+  },
+  "WETH": {
+    "chainId": 1,
+    "address": "0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2",
+    "decimals": 18,
+    "symbol": "WETH",
+    "name": "Wrapped Ether"
+  },
+  "spWETH": {
+    "chainId": 1,
+    "address": "0x59cD1C87501baa753d0B5B5Ab5D8416A45cD71DB",
+    "decimals": 18,
+    "symbol": "spWETH",
+    "name": "Spark WETH"
+  },
```

Comment:
I suggest placing ETH, WETH, and spWETH side by side for better readability and to align with the format of other `data/*.json` files.

Reply:
sure I can align them all. aave v2 groups ETH-like token in the middle and aave v3 has the tokens at the beginning. Which one do you prefer?

Reply:
I don't have a strong preference because, for example `in compound-v3/tokens/data/mainnet.json`, the first one listed is cToken. So, I think it's sufficient as long as the native and wrapped native tokens are putted together.